prow/cmd/deck/static/pr/pr.ts

```diff
@@ -406,10 +406,10 @@ function loadPrStatus(prData: UserData): void {
         let builds: Job[] = [];
         for (let build of allBuilds) {
             if (build.type === 'presubmit' &&
-                build.repo === pr.Repository.NameWithOwner &&
-                build.base_ref === pr.BaseRef.Name &&
-                build.number === pr.Number &&
-                build.pull_sha === pr.HeadRefOID) {
+                build.refs.repo === pr.Repository.NameWithOwner &&
+                build.refs.base_ref === pr.BaseRef.Name &&
+                build.refs.pulls[0].number === pr.Number &&
```

Comment:
Are we certain this array has an entry?

Reply:
I think it's a reasonable assumption (presubmit jobs should probably know the pull request they're referencing), but it probably wouldn't hurt to check anyway.

Reply:
(which would be an improvement over the status quo, where the empty string is an option for filtering by author)